docs/develop/support-ie-11.md

```diff
@@ -54,10 +54,68 @@ if (navigator.userAgent.indexOf("Trident") !== -1) {
 }
 ```
 
+> [!NOTE]
+> Microsoft Edge (Chromium) returns `edg/` followed by one or more version digits and zero or more `.` separators as the user agent; for example, `edg/76.0.167.1`. **Note that the `e` is not present at the end of name! It is "edg", not "edge".**
+
+This JavaScript should be as early in the add-in start up process as possible. The following is an example of an add-in home page that advises users to upgrade Office when Trident is detected.
+
+```html
+<!doctype html>
+<html lang="en" data-framework="typescript">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Contoso Task Pane Add-in</title>
+
+    <!-- Office JavaScript API -->
+    <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
+</head>
+
+<body>
+    <div id="main">
+         <!-- 
+            The add-in UI is here. 
+         -->
+    </div>
+
+    <!-- 
+        The script below makes the following div display if the
+        webview is Trident, and hides the regular div. 
+    -->
+    <div id="tridentmessage" style="display: none; padding: 10;">
+        This add-in will not run in your version of Office. Please upgrade either to 
+        perpetual Office 2021 (or later) or to a Microsoft 365 account.
+    </div>
+    <script>
+        if (navigator.userAgent.indexOf("Trident") !== -1) {
+            var tridentMessage = document.getElementById("tridentmessage");
+            var normalUI = document.getElementById("main");
+            tridentMessage.style.display = "block";
+            normalUI.style.display = "none";
+        } 
+    </script>
+</body>
+</html>
+```
+
 > [!IMPORTANT]
-> It's not usually a good practice to read the `userAgent` property. Be sure you're familiar with the article, [Browser detection using the user agent](https://developer.mozilla.org/docs/Web/HTTP/Browser_detection_using_the_user_agent), including the recommendations and alternatives to reading `userAgent`. In particular, if you're providing an alternate add-in experience to support the use of Trident, consider using feature detection instead of testing for the user agent.
+> It's not always a good practice to read the `userAgent` property. Be sure you're familiar with the article, [Browser detection using the user agent](https://developer.mozilla.org/docs/Web/HTTP/Browser_detection_using_the_user_agent), including the recommendations and alternatives to reading `userAgent`. In particular, if you're providing an alternate add-in experience to support the use of Trident, consider using feature detection instead of testing for the user agent. But if you are just providing a notification that the add-in doesn't work in Trident, as in this case, using `userAgent` is appropriate. 
```

Comment:
```suggestion
> It's not always a good practice to read the `userAgent` property. Be sure you're familiar with the article, [Browser detection using the user agent](https://developer.mozilla.org/docs/Web/HTTP/Browser_detection_using_the_user_agent), including the recommendations and alternatives to reading `userAgent`. In particular, if you're providing an alternate add-in experience to support the use of Trident, consider using feature detection instead of testing for the user agent. But if you're just providing a notification that the add-in doesn't work in Trident, as in this case, using `userAgent` is appropriate. 
```